src/ESPEasy-Globals.h

```diff
@@ -521,10 +521,10 @@ bool showSettingsFileLayout = false;
   #define FILE_RULES        "/rules1.txt"
   #include <WiFi.h>
   #include  "esp32_ping.h"
-  #include <ESP32WebServer.h>
+  #include <WebServer.h>
   #include "SPIFFS.h"
   #include <rom/rtc.h>
-  ESP32WebServer WebServer(80);
+  WebServer WebServer(80);
```

Comment:
I guess for this also the 'lib_deps' in Platformio.ini must be changed for `[core_esp32]` ?
Or is it an alias for ESP32WebServer now?

Reply:
It 's possible, I don't use Platformio

Reply:
The build fails because of this and it complains there is no Webserver.h.

Reply:
core_esp32_0_12_0 does not have the WebServer,  core_esp32_1_2_0 as minimum version will do.
Since this version the additional ESP32WebServer library is obsolete.